### PR TITLE
Feature/nav component

### PR DESCRIPTION
**Pull request by r1g023**
# Description
Issue # (add the issue number here, if applicable)

Describe what you're adding

- only testing Flex with the Nav bar within the Landing page
- more testing needed, will keep updating this branch until the nav bar is fully done 

# QA
List out how to test your changes and add a screenshot of what it should look like, when applicable.

### CODE REVIEW

**Comment by r1g023** [comment]
@zeeberry I've finished the Nav component and added it within the Landing page

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (zeeberry) [inline .vscode/settings.json:1]: we dont want to commit this to the repo. we may have to add this to the .gitignore

File: src/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <Square bg="blue.500" size="150px">
+        <Text>Box 2</Text>
+      </Square>
+      <Box flex="1" bg="tomato">
+        <Text>Box 3</Text>
+      </Box>
+    </Flex>
+  );
+}
+
```
Comment (zeeberry) [inline src/components/Navigation.js:34]: nice use of chakra

File: src/components/LandingPage.js [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
+              "leftUpperCard main "
+              "leftBottomCard main  "
+              `}
+      gridTemplateRows={"4.1% 2fr 50% 0px"}
+      gridTemplateColumns={"27% 1fr"}
+      color="blackAlpha.700"
+      fontWeight="bold"
+      w={"100wh"}
+      h={"100vh"}>
+      {/* Navigation */}
```
Comment (zeeberry) [inline src/components/LandingPage.js:19]: do we need this comment?

**Review by zeeberry — COMMENTED** [review]
Left a couple comments. Overall it looks good. we just dont want to merge in the vscode config stuff. Let me know once you remove it so I can approve this PR

**Comment by zeeberry** [comment]
also it'd be great to see a screenshot in the QA section to see what these changes would look like -- just a reminder for next time.

File: src/components/Navigation.js (hunk 2 above)
Comment (r1g023) [reply src/components/Navigation.js:34]: Thank you Z

File: .vscode/settings.json (hunk 1 above)
Comment (r1g023) [reply .vscode/settings.json:1]: Ooops, I have added .vscode to the .gitignore file and pushed changes under PR https://github.com/Category-Colors/category-colors/pull/31

File: src/components/LandingPage.js (hunk 3 above)
Comment (r1g023) [reply src/components/LandingPage.js:19]: Thank you for the feedback, I have removed all comments on LandingPage and pushed changes to PR https://github.com/Category-Colors/category-colors/pull/31

**Review by zeeberry — APPROVED** [review]

**Pull request merged**